src/datastore/postgres/PgDatastore.ts

```diff
@@ -152,15 +152,23 @@ export class PgDatastore implements Datastore {
     }
 
     public async getEventBySlackId(slackChannel: string, slackTs: string): Promise<EventEntry|null> {
-        return this.postgresDb.oneOrNone(
+        log.debug(`getEventBySlackId: ${slackChannel} ${slackTs}`);
+        const events = await this.postgresDb.manyOrNone(
```

Comment:
To me the significant change isn't the await (which introduces no change as far as I can tell, but the change from `oneOrNone` to `manyOrNone`.
This does not seem to fix the root of the issue that we have one event stored multiple times in the database.

Reply:
So the problem really is that we have multiple matrix events for one slack event. The problem comes in because we end up with two results if we search by slackTs. This PR is more of an attempt to do a best-effort fetch the first event rather than catch-fire-and-fail.

Ideally, we wouldn't store multiple events.